posthog/temporal/data_imports/workflow_activities/create_job_model.py

```diff
@@ -46,7 +46,7 @@ async def create_external_data_job_model_activity(inputs: CreateExternalDataJobM
         database = source.job_inputs.get("database")
         db_schema = source.job_inputs.get("schema")
 
-        using_ssh_tunnel = source.job_inputs.get("ssh_tunnel_enabled")
+        using_ssh_tunnel = str(source.job_inputs.get("ssh_tunnel_enabled", False)) == "True"
```

Comment:
question(non-blocking):

> Job inputs always return strings

Saw that `job_inputs` field it's encoded as JSON which means we can serialize booleans. Where's the limitation coming from that only strings are supported?

Reply:
Great question, i'm unsure - something we could likely do for sure

Reply:
Thank you! It would certainly look cleaner here. But it's something we can tackle in a follow-up PR, this should work for now.